dart/lib/src/protocol/sentry_event.dart

```diff
@@ -26,8 +30,6 @@ class SentryEvent {
     this.message,
     this.transaction,
     dynamic throwable,
-    this.stackTrace,
```

Comment:
why is stack trace removed?

Reply:
oh maybe because when building the event and `attachStackTrace` is enabled, we can add it directly to the thread list instead of during serialization

Reply:
Exactly